ui-components/src/forms/Textarea.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react"
+import { Textarea } from "./Textarea"
+
+export default {
+  title: "Text/Textarea",
```

Comment:
Can we relocate this to the "Forms" area of Storybook?

Reply:
It lives under src/forms? Not sure what you mean can you clarify?